R/pkgs/report.generation/R/setup_testing_environment.R

```diff
@@ -0,0 +1,221 @@
+#' @export
+setup_testing_environment <- function(cf = "config.yml"){
```

Comment:
Didn't we delete this forever and ever and ever

Reply:
I think this is a question for @juanderone ... I didn't edit this. maybe this is an accidental carryover from an older branch